src/cli/package_manager_command.zig

```diff
@@ -92,6 +93,7 @@ pub const PackageManagerCommand = struct {
             \\
             \\<b>Commands:<r>
             \\
+            \\  <b><green>bun pm<r> <blue>scan<r>                 scan all packages in lockfile for security vulnerabilities
```

Comment:
isn't this going to confuse everyone with what it does compared to `bun audit`, also "scan" to me doesn't sound like its only for security